slides/02-data-budget.qmd

```diff
@@ -219,33 +219,34 @@ We will use this tomorrow
 
 # Exploratory data analysis for ML 🧐
 
-## 
+## Your turn {transition="slide-in"}
+
+![](images/parsnip-flagger.jpg){.absolute top="0" right="0" width="150" height="150"}
+
+*Explore the `frog_train` data on your own!*
+
+* *What's the distribution of numeric variables like age?*
+* *How does latency differ across the categorical variables?*
 
 ```{r}
-#| fig-align: 'center'
-ggplot(frog_train, aes(latency)) +
-  geom_histogram(bins = 20)
+#| echo: false
+countdown(minutes = 8, id = "explore-frogs")
 ```
 
-. . .
-
-Remember how we used `strata = latency`?
+::: notes
+Make a plot or summary and then share with neighbor
+:::
 
 ## 
 
 ```{r}
 #| fig-align: 'center'
-quartiles <- quantile(frog_train$latency, probs = c(1:3)/4)
 ggplot(frog_train, aes(latency)) +
-  geom_histogram(bins = 20) +
-  geom_vline(xintercept = quartiles, color = train_color, 
-             size = 1.5, lty = 2)
+  geom_histogram(bins = 20)
```

Comment:
Do you want to add a speaker note to this slide to _hint_ at the fact that we might need to stratify by latency to preserve the distribution? That hint motivates the "split smarter" section so id hate to forget to say it here